.github/CODEOWNERS

```diff
@@ -9,3 +9,4 @@
 /pg_bm25/                                                      @rebasedming @neilyio
 /pg_sparse/                                                    @rebasedming @sardination
 /shared/                                                       @rebasedming @neilyio
+/tokenizers/                                                   @rebasedming @neilyio
```

Comment:
We might also want to add a path in `test-pg_bm25.yml` so that if there are changes to `tokenizers/` it gets triggered as well. Otherwise they'll slip under the CI radar